components/shared/README.md

```diff
@@ -0,0 +1,40 @@
+# browserd/shared
+
+Represents some functionality that is shared between the consumer and provider 🤕☁✨
+
+[![Build Status](https://dev.azure.com/bengreenier/browserd/_apis/build/status/shared?branchName=master)](https://dev.azure.com/bengreenier/browserd/_build/latest?definitionId=12&branchName=master)[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=bengreenier_browserd&metric=alert_status)](https://sonarcloud.io/dashboard?id=bengreenier_browserd)
+
+## Testing
+
+How to test shared functinoality. ⚙
+
+### Docker
+> Note: Our [`Dockerfile`](./Dockerfile) has a defined display resolution of `1920x1080` - You may change this as needed.
+
+build the container (and source)
+```
+docker build . -f components/shared/DOCKERFILE -t browserd:local
```

Comment:
do you think docker still makes sense to have here?

Reply:
I was thinking that since it works as a testing method I might as well include it, but I suppose it is a bit overkill